### DISCUSSION TITLE
Invalid username or password for 4.6.1

### DISCUSSION THREAD

**Discussion by sirzur** [userdocs/qbittorrent-nox-static#128] [General]
I downloaded the libtorrent 1.2.19 qt5 version of qbittorrent-nox-static v4.6.1, armhf.  But when I ran it, I could see the program started but I could not log into the GUI because of incorrect username password combination.  Replacing the v4.6.1 with v4.6.0 or v 4.5.5 does not have that issue.  Has the password changed?

This is the file I downloaded "armhf-qt5-qbittorrent-nox".

By the way, should this be here or in discussion?

**Comment by userdocs** [comment]
https://github.com/userdocs/qbittorrent-nox-static/discussions/126

**Comment by sirzur** [comment]
OK.  Looks like an upstream qbittorrent requirement to change the default username/password combination.  With 4.6.0 installed, I changed the password for the GUI and confirmed new password in the "WebUI\Password_PBKDF2=" entry of the configuration file.  I then updated to v4.6.1 and the new password works
